public/admin/config.yml

```diff
@@ -115,11 +115,6 @@ meta_config: &meta_config
       widget: "object"
       collapsed: true
       fields:
-        - name: "title"
-          label: "Twitter Title"
-          widget: "string"
-          default: "summary"
-          required: false
```

Comment:
Is it not needed?

Reply:
If open graph metadata and twitter metadata are optional, than we can also make this clear for the parent object through the `config.yml`

![CleanShot 2024-09-25 at 11 21 01@2x](https://github.com/user-attachments/assets/646bfbe8-719b-4d5d-b23a-d665ce24e52d)
![CleanShot 2024-09-25 at 11 21 40@2x](https://github.com/user-attachments/assets/1161f9ce-ec6b-4a5a-b99f-4481c867aab6)

Reply:
Good thinking! As for twitter title - it will default to OG title. One less thing for them to think about, but if needed we can add if they want to customise it?